frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/base/link_to_page_widget.dart

```diff
@@ -44,13 +44,15 @@ void showLinkToPageMenu(
             await editorState.insertReferencePage(viewPB, pageType);
             linkToPageMenuEntry.remove();
           } on FlowyError catch (e) {
-            Dialogs.show(
-              child: FlowyErrorPage.message(
-                e.msg,
-                howToFix: LocaleKeys.errorDialog_howToFixFallback.tr(),
-              ),
-              context,
-            );
+            if (context.mounted) {
+              Dialogs.show(
+                child: FlowyErrorPage.message(
+                  e.msg,
+                  howToFix: LocaleKeys.errorDialog_howToFixFallback.tr(),
+                ),
+                context,
```

Comment:
I know this was not introduced in this PR, but this seems odd to me, that a named parameter is before a positional, and the named parameter is `child` but isn't sorted last.

Reply:
I know, but I don't want to include this change in this PR since it has a lot of changes that will conflict with the main branch easily.